contracts/network/BancorNetwork.sol

```diff
@@ -902,6 +902,10 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         bool isMigrating,
         uint256 originalAmount
     ) private returns (uint256) {
+        if (msg.value > 0) {
```

Comment:
better not access msg.sender in a non external method, can have side effects if the logic changes.

Reply:
nm, confused with msg.sender